webcrypto/goja.go

```diff
@@ -29,6 +33,34 @@ func exportArrayBuffer(rt *goja.Runtime, v goja.Value) ([]byte, error) {
 	return ab.Bytes(), nil
 }
 
+// traverseObject traverses the given object using the given fields and returns the value
+// at the end of the traversal. It assumes that all the traversed fields are Objects.
+func traverseObject(rt *goja.Runtime, src *goja.Object, fields ...string) (goja.Value, error) {
+	var value goja.Value = src
+	var currentField string
+	var obj *goja.Object
+
+	for idx, field := range fields {
+		if idx > 0 {
+			currentField += "."
+		}
+
+		currentField += field
+
+		obj = value.ToObject(rt)
+		if isNullish(obj) {
```

Comment:
I don't think this is possible at all - I am pretty sure `ToObject` panics in those cases, and we do check the value after we get it in the line below.

It might be worth moving this to above the `for` for the input `value` especially if we refactor it to be taking a value instead of an object.